wolfcrypt/src/port/iotsafe/iotsafe.c

```diff
@@ -1050,9 +1172,14 @@ static int wolfIoT_ecc_shared_secret(WOLFSSL* ssl, struct ecc_key* otherKey,
             if (ret > 0) {
                 /* Generating shared secret
                 */
+#if (IOTSAFE_ID_SIZE == 1)
```

Comment:
There are already places in the code iotsafe.h lines 86-99 where IOTSAFE_ID_SIZE is checked for 1 or 2. Does this need to be here also?